src/it/scala/com/weightwatchers/reactive/kinesis/common/AkkaUnitTestLike.scala

```diff
@@ -1,14 +1,28 @@
 package com.weightwatchers.reactive.kinesis.common
 
 import akka.actor.{ActorSystem, Scheduler}
-import akka.stream.{ActorMaterializer, Materializer}
 import akka.testkit.TestKitBase
 import com.typesafe.config.{Config, ConfigFactory}
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{BeforeAndAfterAll, Suite}
-
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.freespec.AnyFreeSpecLike
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.{BeforeAndAfterAll, GivenWhenThen, Suite}
+import org.scalatestplus.mockito.MockitoSugar
 import scala.concurrent.ExecutionContextExecutor
 
+
+/**
+  * Base trait for all integration tests.
+  */
+trait IntegrationTest
```

Comment:
As a side question. Can we use the https://github.com/moia-dev/itkit here?

Reply:
This is a fork from https://github.com/WW-Digital/reactive-kinesis and open source. 
itkit is private. So no we can not use it here.